packages/mdc-feature-targeting/README.md

```diff
@@ -0,0 +1,74 @@
+<!--docs:
+title: "Feature Targeting"
+layout: detail
+section: components
+excerpt: "Provides infrastructure to allow CSS styles to be included or excluded categorically."
+path: /catalog/feature-targeting/
+-->
+
+# Feature Targeting
+
+MDC Feature Targeting provides infrastructure to allow CSS styles to be included or excluded categorically.
+
+Most of the time, you shouldn't need to depend on `mdc-feature-targeting` directly. However, understanding it can be useful if you're interested in having more control over when certain types of MDC styles are emitted.
+
+## Installation
+
+```
+npm install @material/feature-targeting
+```
+
+## Basic Usage
+
+### Styles
+
+Authoring component styles:
+
+```scss
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
+
+@mixin my-component-core-styles($query: mdc-feature-all()) {
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @include mdc-feature-targets($feat-structure) {
+    // ...
+  }
+}
+```
+
+Consuming component styles:
+
+```scss
+@import "my-component-mixins";
+
+// To include all styles (using the default of mdc-feature-all() defined above):
+@include my-component-core-styles;
+
+// Or, to include a specific subset of styles:
+@include my-component-core-styles(mdc-feature-any(structure));
```

Comment:
This query can be expressed more simply as just `@include my-component-core-styles(structure)`

Reply:
I simplified this and added more examples to show more function usage cases.